File: features/list/src/main/java/com/vp/list/model/ListItem.java
package com.vp.list.model;

import com.google.gson.annotations.SerializedName;

public class ListItem {
    @SerializedName("Title")
    private String title;
    @SerializedName("Year")
    private String year;
    private String imdbID;
    @SerializedName("Poster")
    private String poster;

    public String getTitle() {
        return title;
    }

    public String getYear() {
        return year;
    }

    public String getImdbID() {
        return imdbID;
    }

    public String getPoster() {
        return poster;
    }

    public void setImdbID(String imdbID) {
        this.imdbID = imdbID;
    }

    public void setPoster(String poster) {
        this.poster = poster;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public void setYear(java.lang.String year) {
        this.year = year;
    }
}

File: features/database/src/main/java/com/vp/database/model/entity/ListItem.kt
package com.vp.database.model.entity

import com.google.gson.annotations.SerializedName

class ListItem(@SerializedName("Title")
               var title: String = "",
               @SerializedName("Year")
               var year: String = "",
               var imdbID: String = "",
               @SerializedName("Poster")
               var poster: String = "")
